src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsOnSpecialOasis.java

```diff
@@ -0,0 +1,81 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.contentpatterns.AdsContent;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+
+import org.openqa.selenium.Dimension;
+import org.testng.annotations.Test;
+
+/**
+ * https://www.google.com/dfp/5441#delivery/LineItemDetail/lineItemId=126608052
+ */
+
+public class TestAdsOnSpecialOasis extends TemplateNoFirstLoad {
+  final static String PROJECT43_TEST_LINE_ITEM_ID = "271491732";
+
+  @Test(groups = "TestAdsOnSpecialPagesOasis")
+  public void testAdsOnSpecialVideoPageOasis() throws Exception {
+    AdsBaseObject ads = buildAdsObjectForPage("Special:Videos");
+    verifySlotHasCorrectAd(
+        ads,
+        AdsContent.TOP_LB,
+        "wka.life/_project43//special",
+        PROJECT43_TEST_LINE_ITEM_ID
+    );
+  }
+
+  @Test(groups = "TestAdsOnSpecialPagesOasis")
+  public void testAdsOnSpecialImagesPageOasis() throws Exception {
+    AdsBaseObject ads = buildAdsObjectForPage("Special:Images");
+    verifySlotHasCorrectAd(
+        ads,
+        AdsContent.TOP_LB,
+        "wka.life/_project43//special",
+        PROJECT43_TEST_LINE_ITEM_ID);
+  }
+
+  private AdsBaseObject buildAdsObjectForPage(String pageName) {
+    String testedPage = urlBuilder.getUrlForPath("project43", pageName);
+    return new AdsBaseObject(driver, testedPage, new Dimension(1292, 1000));
+  }
+
+  private void verifySlotHasCorrectAd(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "verifySlotHasCorrectAd" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)